keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/constraints/StatefulConstraintEvaluator.kt

```diff
@@ -24,10 +24,16 @@ import com.netflix.spinnaker.keel.api.events.ConstraintStateChanged
 /**
  * TODO: Docs.
  */
-abstract class StatefulConstraintEvaluator<T : Constraint>(
+abstract class StatefulConstraintEvaluator<T : Constraint, A : ConstraintStateAttributes>(
```

Comment:
Can you please explain this change? I don't really understand what is happening here